feat(content): Patir 2 (#9959)
endless-sky/endless-sky: data/kahet/beyond patir.txt

		"acceleration" .0000001
		"drag" 1
		"lifetime" 10
		"submunition" "asteroid missile"

outfit "creeper15 print"
	weapon
		sprite "effect/asteroid projectile print"
			"start frame" 1
			"frame rate" 9
		"velocity" 2
		"acceleration" 3
		"drag" .1
		"turn" -8
		"lifetime" 5
		"submunition" "creeper15 static"
		
outfit "creeper15 static"
	weapon
		sprite "effect/asteroid projectile static"
			"start frame" 1
			"frame rate" 9
		"acceleration" .0000001
		"drag" 1
		"lifetime" 5
		"submunition" "asteroid missile"

outfit "asteroid missile"
	weapon
		sprite "effect/asteroid projectile print"
			"start frame" 1
			"frame rate" 9
		"acceleration" 2
		"drag" .1
		"turn" 6
		"homing" 3
		"tracking" 1
		"lifetime" 120

outfit "Asteroid Weapon Large 2"
	category "Guns"
	cost 71670239
	thumbnail "outfit/asteroid weapon 1"
	"mass" 31
	"outfit space" -31
	"weapon capacity" -31
	"gun ports" -1
	"asteroid mount 2" -1
	weapon
		"velocity" 10
		"drag" .1
		"acceleration" 1
		"lifetime" 200000
		"submunition" "asteroid laser 1"
			"offset" 0 10
		"submunition" "spin1 print right"
		"submunition" "spin1 print left"
		"fire effect" "asteroid glow"
		"split range" 180
		"reload" 150
		"firing heat" 5

> 	description "A very peculiar weapon, capable of creating a whole army around a target armed with a laser weapon that consumes its host to fire. However, the precision it requires as well as the raw damage output limits its potential."

Author: Not exactly - this is the effect, for reference (I'll attack it to the OP shortly)

https://github.com/endless-sky/endless-sky/assets/45330165/c0c65708-4618-4a81-bc89-ed2176dad488

> 	description "A very peculiar weapon, capable of creating a whole army around a target armed with a laser weapon that consumes its host to fire. However, the precision it requires as well as the raw damage output limits its potential."

Reviewer: ```suggestion
	description "This very peculiar weapon overwhelms its target by creating an array of offensive entities, each armed with laser weapons that consume itself to fire. However, the precision it requires, as well as its low raw damage output, limits its potential."
```
Something like this is better?

> 	description "A very peculiar weapon, capable of creating a whole army around a target armed with a laser weapon that consumes its host to fire. However, the precision it requires as well as the raw damage output limits its potential."

Reviewer: ```suggestion
	description "This very peculiar weapon overwhelms its target by creating an array of short-living, laser-armed entities. However, the precision it requires, as well as its low raw damage output, limits its potential."
```

			"frame rate" 5
			"random start frame"
		"velocity" -25
		"random velocity" 20
		"inaccuracy" 5
		"acceleration" -1
		"live effect" "asteroid circle trail" 5
		"drag" .1
		"turn" 5
		"lifetime" 40
		"submunition" "track4 print left"

outfit "track4 print right"
	weapon
		sprite "projectile/asteroid weapon swarm"
			"frame rate" 5
			"random start frame"
		"acceleration" 1
		"drag" .1
		"turn" 20
		"live effect" "asteroid circle trail" 5
		"homing" 3
		"tracking" 1
		"lifetime" 550
		"inaccuracy" 5
		"split range" 180
		"submunition" "asteroid laser 1"
			"offset" 0 10

outfit "track4 print left"
	weapon
		sprite "projectile/asteroid weapon swarm"
			"frame rate" 5
			"random start frame"
		"acceleration" 1
		"drag" .1
		"turn" 20
		"live effect" "asteroid circle trail" 5
		"homing" 3
		"tracking" 1
		"lifetime" 550
		"inaccuracy" 5
		"split range" 180
		"submunition" "asteroid laser 1"
			"offset" 0 10

outfit "asteroid laser 1"
	weapon
		sprite "projectile/asteroid laser"
			"frame rate" 60
			"random start frame"
		sound "zapper"
		"hit effect" "skylance impact"
		"target effect" "asteroid target"
		"inaccuracy" .3
		"velocity" 200
		"lifetime" 1
		"reload" 1

> 		"firing heat" 1.4

Reviewer 2: If this only ever exists as a submunition, then firing heat won't do anything.

Author: Removed

		"discharge damage" 1.7
		"hull damage" 5.9

effect "asteroid hit"
	sprite "effect/point defense hit"
		"frame rate" 13
		"no repeat"
	lifetime 1
	"velocity scale" 0

effect "asteroid target"
	sprite "effect/finisher sparks"
		"random start frame"
		"frame rate" 20
	"lifetime" 15
	"random angle" 180

effect "asteroid circle trail"
	sprite "effect/asteroid circle trail"
		"no repeat"
		"frame time" 4
	"lifetime" 20
	"random velocity" 5
	"random spin" 10
	"velocity scale" 0.3

outfit "Asteroid Weapon Large 3"
	category "Guns"
	cost 98713449
	thumbnail "outfit/asteroid weapon 5"
	"mass" 16
	"outfit space" -16
	"weapon capacity" -16
	"gun ports" -1
	"asteroid mount 3" -1
	weapon
		sprite "projectile/asteroid twin projectile"
			"frame rate" 10
			"random start frame"
		"velocity" 20
		"drag" .1
		"acceleration" 1
		"lifetime" 30
		"submunition" "asteroid missile"
			"facing" 30
		"submunition" "asteroid missile"
			"facing" -30
		"submunition" "magic deployer1"
		"homing" 1
		"optical tracking" 1
		"turn" 0
		"split range" 180
		"reload" 100
		"firing heat" 5

> 	description "While it may be technically described as a missile, its specifics differ from the average rocket launcher. The heap of energy it releases continues shooting enemy-seeking bullets for some time, before finally exploding into a barrage of those same projectiles. Great for overthrowing an anti-missile, and impact a fair share of damage in the meantime"

Reviewer: ```suggestion
	description "While it may be technically described as a missile, its specifics differ from the average rocket launcher. The heap of energy created by this weapon's fire continuously releases projectiles which home in onto enemies, until finally exploding and breaking up into an enormous barrage of the same projectiles. These attributes make it very good at overwhelming anti-missiles enough to render them useless, then impacting a fair share of damage while they are down."
```
Again, not sure on how accurate the rewrite is.

		"drag" .1
		"acceleration" 1
		"lifetime" 30
		"submunition" "asteroid missile" 15
			"facing" 30
		"submunition" "asteroid missile" 15
			"facing" -30

outfit "Asteroid Jump Drive"
	category "Systems"
	cost 29631456
	thumbnail "outfit/asteroid weapon 7"
	"mass" 20
	"outfit space" -20
	"jump speed" 0.2
	"jump fuel" 200
	"jump drive" 1
	"asteroid mount jd" -1
	"jump effect" "jump drive circle"
	"jump effect" "jump drive spark 1"
	"jump effect" "jump drive spark 2"
	"jump effect" "jump drive spark 3"
	"jump effect" "jump drive spark 4"
	"jump effect" "jump drive spark 5"
	"jump effect" "jump drive spark 6"
	"jump effect" "jump drive spark 7"
	"jump effect" "jump drive spark 8"
	"jump effect" "jump drive spark 9"
	"jump effect" "jump drive 1"
	"jump effect" "jump drive 2"
	"jump effect" "jump drive 3"
	"jump effect" "jump drive 4"
	"jump effect" "jump drive 5"
	"jump effect" "jump drive 6"
	"jump effect" "jump drive 7"
	"jump effect" "jump drive 8"
	"jump effect" "jump drive 9"
	"jump effect" "jump drive 10"
	"jump effect" "jump drive 11"
	"jump effect" "jump drive 12"
	"jump effect" "jump drive 13"
	"jump effect" "jump drive 14"
	"jump effect" "jump drive 15"

> 	description "Whatever the mechanism that hides beneath the translucent orb may be, its function is clear: a Jump Drive, connecting the different parts of the universe together for the mere cost of 200 fuel units."

Reviewer: ```suggestion
	description "Whatever the mechanism that hides beneath the translucent orb may be, it functions similarly to a jump drive, bridging different parts of the universe together for the cost of merely 200 units of fuel."
```

		"frame time" 6
		"random start frame"
	"lifetime" 15
	"random spin" 3
	"random angle" 180

effect "jump drive spark 6"
	sprite "effect/repulsor impact"
		"frame time" 6
		"random start frame"
	"lifetime" 15
	"random spin" 3
	"random angle" 180

effect "jump drive spark 7"
	sprite "effect/repulsor impact"
		"frame time" 6
		"random start frame"
	"lifetime" 15
	"random spin" 3
	"random angle" 180

effect "jump drive spark 8"
	sprite "effect/repulsor impact"
		"frame time" 6
		"random start frame"
	"lifetime" 15
	"random spin" 3
	"random angle" 180

effect "jump drive spark 9"
	sprite "effect/repulsor impact"
		"frame time" 6
		"random start frame"
	"lifetime" 15
	"random spin" 3
	"random angle" 180

outfit "Asteroid Weapon Large 4"
	category "Guns"
	cost 57165239
	thumbnail "outfit/asteroid weapon 4"
	"mass" 33
	"outfit space" -33
	"weapon capacity" -33
	"gun ports" -1
	"asteroid mount 4" -1
	weapon
		sprite "effect/asteroid projectile print"
			"frame rate" 21
		"velocity" 100
		"drag" .1
		"acceleration" 1
		"lifetime" 20
		"turn" -6
		"submunition" "wep1 print" 2
			"facing" 15
		"reload" 110
		"burst count" 7
		"burst reload" 1
		"firing heat" 5

> 	description "An impressive defensive weapon, scaring off even the mightiest attackers by projecting enormous bubble-shaped mine fields all around itself. Though not rapid firing by any mean, it can continuously maintain a field without issues."

Reviewer: ```suggestion
	description "An impressive defensive weapon, fending off even the mightiest attackers by projecting enormous bubble-shaped minefields all around itself. Though not a rapid-fire weapon by any means, it is able to maintain one of those fields continuously without issue."
```

# Foundation, either version 3 of the License, or (at your option) any later version.
#
# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
# PARTICULAR PURPOSE. See the GNU General Public License for more details.
#
# You should have received a copy of the GNU General Public License along with
# this program. If not, see <https://www.gnu.org/licenses/>.

phrase "magic asteroid"
	word
		"???"

government "Magic Asteroid"
	"display name" "???"
	swizzle 0
	language "???"
	
	"attitude toward"
		"Ka'sei (12A)" -.01
		"Uninhabited" -.01

	"player reputation" 10
	"friendly hail" "friendly ka'het"
	"friendly disabled hail" "friendly disabled ka'het"
	"hostile hail" "hostile ka'het"
	"hostile disabled hail" "hostile disabled ka'het"

government "Ka'sei (12A)"
	"display name" "Ka'sei"
	swizzle 0
	language "???"
	
	"penalty for"
		assist 0
		disable 0
		board 0
		capture 0
	"player reputation" 10
	"friendly hail" "friendly ka'het"
	"friendly disabled hail" "friendly disabled ka'het"
	"hostile hail" "hostile ka'het"

> 	"hostile disabled hail" "hostile disabled ka'het"

Author: Moved to governments.txt

		"submunition" "wep static"
		"submunition" "wep40 print"
	
outfit "wep40 print"
	weapon
		sprite "effect/asteroid projectile print"
			"start frame" 1
			"frame rate" 9
		"velocity" 2
		"acceleration" 3
		"drag" .1
		"turn" -2
		"lifetime" 15
		"random lifetime" 5
		"submunition" "wep static"
		"submunition" "wep41 print"
	
outfit "wep41 print"
	weapon
		sprite "effect/asteroid projectile print"
			"start frame" 1
			"frame rate" 9
		"velocity" 2
		"acceleration" 3
		"drag" .1
		"turn" -2
		"lifetime" 15
		"random lifetime" 5
		"submunition" "wep static"
		"submunition" "wep42 print"
	
outfit "wep42 print"
	weapon
		sprite "effect/asteroid projectile print"
			"start frame" 1
			"frame rate" 9
		"velocity" 2
		"acceleration" 3
		"drag" .1
		"turn" -2
		"lifetime" 15
		"random lifetime" 5
		"submunition" "wep static"

outfit "wep static"
	weapon
		sprite "effect/asteroid projectile static"
			"start frame" 1
			"frame rate" 9
		"acceleration" .0000001
		"drag" 1
		"lifetime" 575
		"turn" 360
		"homing" 2
		"tracking" 1
		"trigger radius" 240
		"blast radius" 400
		"discharge damage" 1.7
		"hull damage" 72.9
		"slowing damage" 10
		"hit force" 150
		"hit effect" "circle defense"

> effect "circle defense"

Author: Renamed to "beyond patir assets"
